src/BizHawk.Client.EmuHawk/tools/TAStudio/MarkerControl.cs

```diff
@@ -254,6 +265,37 @@ public void EditMarkerPopUp(TasMovieMarker marker, bool followCursor = false)
 			}
 		}
 
+		public void EditMarkerFramePopUp(TasMovieMarker marker, bool followCursor = false)
+		{
+			var markerFrame = marker.Frame;
+			var i = new InputPrompt
+			{
+				Text = $"Marker for frame {markerFrame}",
+				TextInputType = InputPrompt.InputType.Unsigned,
+				Message = "Enter a frame number",
+				InitialValue =
+					Markers.IsMarker(markerFrame)
+					? Markers.PreviousOrCurrent(markerFrame).Frame.ToString()
+					: "0"
+			};
+
+			if (followCursor)
+			{
+				var point = Cursor.Position;
+				point.Offset(i.Width / -2, i.Height / -2);
+				i.StartPosition = FormStartPosition.Manual;
+				i.Location = point;
+			}
+
+			if (this.ShowDialogWithTempMute(i) == DialogResult.OK)

```

Comment:
To match behavior of 

https://github.com/TASEmulators/BizHawk/blob/3ae32082b2f40ae20a0a474397cf25780a40ddf7/src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.ListView.cs#L603

refuse to move the marker if there's already a marker on the target frame. Having 2 markers on the same frame is **bad**.